### ISSUE TITLE
Release PR isn't triggering any status check

### ISSUE THREAD

**Issue by matteo-rossi-wise** [google-github-actions/release-please-action#539]
### TL;DR

When a Release PR is automatically created, I'd expect any other workflow whose trigger is PR creation (e.g. `opened`, `reopened`, `synchronize`) to be run and listed within the PR `Checks` section. But no check is run.

### Expected behavior

When a Release PR is automatically created, I'd expect any other workflow whose trigger is PR creation (e.g. `opened`, `reopened`, `synchronize`) to be run and listed within the PR `Checks` section.

As an example, we use Dependabot to automatically bump our gems on a scheduled basis. When a Dependabot PR is opened, all the workflows running on a PR event are correctly run.

For instance, we have a Test workflow (.github/workflows/test.yml) that is run on PR creation or when a PR is updated.

### Observed behavior

With the `release-please` action, none of the workflows, whose trigger is a PR creation or synchronization, is run.

### Action YAML

```yaml
name: Release app
inputs:
  github-token:
    description: A GitHub Token
    required: true
runs:
  using: composite
  steps:
    - uses: actions/checkout@v3
      id: checkout
      with:
        token: ${{inputs.github-token}}
    - uses: google-github-actions/release-please-action@v3
      id: release
      with:
        release-type: ruby
        package-name: release-please-action
        changelog-types: '[{"type":"feat","section":"Features","hidden":false},{"type":"fix","section":"Bug Fixes","hidden":false},{"type":"refactor","section":"Refactoring","hidden":false},{"type":"build","section":"Miscellaneous","hidden":false},{"type":"docs","section":"Documentation","hidden":false},{"type":"test","section":"Test","hidden":false}]'
      if: ${{steps.checkout.outcome == 'success'}}
```

**Comment by chingor13** [comment]
Pull requests created/modified by the github-actions built-in user/credentials will not trigger any actions workflows. This is a limitation from the GitHub side and is to prevent infinite loops of action runs.

You will need to configure a personal access token to use in the `token` argument

**Comment by steveoh** [comment]
https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#workarounds-to-trigger-further-workflow-runs

This has some other "better" options than using a PAT
